bin/runtime-common/src/messages.rs

```diff
@@ -99,12 +99,52 @@ pub struct MessageTransaction<Weight> {
 	pub size: u32,
 }
 
+/// Helper trait for estimating the size and weight of a single message delivery confirmation
+/// transaction.
+pub trait TransactionEstimation<Weight> {
+	fn estimate_delivery_confirmation_transaction() -> MessageTransaction<Weight>;
```

Comment:
Could you, please, add some docs for this public method and for the public `BasicTransactionEstimation` struct?

Reply:
Done